packages/adapters/src/adapters/Cornerstone3D/MeasurementReport.ts

```diff
@@ -132,25 +136,21 @@ export default class MeasurementReport {
         return _meta;
     }
 
-    static generateDerivationSourceDataset(
-        StudyInstanceUID,
-        SeriesInstanceUID
-    ) {
+    static generateDerivationSourceDataset = instance => {
```

Comment:
Use the actual instance data for the derived data, as it is used for generating the overall report.